tools/spec2vec/spec2vec_training.xml

```diff
@@ -0,0 +1,254 @@
+<tool id="spec2vec_training" name="Spec2Vec Model Training" version="@TOOL_VERSION@-@TOOL_DEV_VERSION@+galaxy0" python_template_version="3.5" profile="21.05">
+    <description>Train a Spec2Vec model for mass spectra similarity scoring</description>
+
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="creator"/>
+
+    <requirements>
+        <container type="docker">recetox/spec2vec:@COMMIT_SHA@</container>
+    </requirements>
+
+    <command detect_errors="exit_code"><![CDATA[
+        sh ${weights_symlink} &&
+        sh ${spec2vec_python_cli}
+    ]]></command>
+        
+    <configfiles>
+        <configfile name="weights_symlink">
+            ln -fs '${weights_filename}' '${weights_filename}.npy'
```

Comment:
Is there any reason why you use a configfile here and not include this directly in the command section?

Reply:
It appears to me that storing symlinks in a config file would improve code readability.

Reply:
After more thoughts on this, using one oneliner instead of another oneliner doesn't seem like much of a readability improvement. Moved symlinks to command in 4e0fab4.

Reply:
For small code like this it also helps admins later I assume. Now the command ends up in the main tool_script.sh file and it should be obvious what's happening for an Galaxy admin. With a config file this is partially hidden in an additional file.

Reply:
That's a good point!